README.md

```diff
@@ -1,7 +1,8 @@
-# The orthw script
+# The ORT Helper Wrapper (`orthw`) script
 
 `orthw` is a shell script designed to simplify and speed up common tasks performed when processing [ORT][ort]
-scan results. It supports a number of commands, which can be run in the terminal (e.g. Bash). 
+scan results. It supports a number of commands, which can be run in the terminal (e.g. Bash).
+Most of the commands are thin wrappers around the commands provided by ORT's helper CLI. 
 
```

Comment:
Propose to make ORT's helper CLI a link to https://github.com/oss-review-toolkit/ort/tree/main/helper-cli or better would be if we had a docs page for the helper CLI (working on this). My intent is that we clarify what ORT Helper is..